chef/cookbooks/provisioner/recipes/base.rb

```diff
@@ -350,7 +350,14 @@
 address = crowbar_node["crowbar"]["network"]["admin"]["address"]
 protocol = crowbar_node["crowbar"]["apache"]["ssl"] ? "https" : "http"
 server = "#{protocol}://#{address}"
-password = crowbar_node["crowbar"]["users"]["crowbar"]["password"]
+is_admin = CrowbarHelper.is_admin?(node)
+if is_admin
+  username = "crowbar"
+  password = crowbar_node["crowbar"]["users"][username]["password"]
```

Comment:
Not really something new (as the bug was already there): we should check that there's a crowbar user, and if not, take another user. Nothing prevents the cloud admin to change usernames...